magento/pwa-studio
# MagentoRouteHandler: correct handling of unknown routes

## Description
MagentoRouteHandler now correctly returns NotFound if the urlResolver-Query returns null.
Further description see issue #1454 

## Related Issue
Closes #1454 .

## Verification Steps
1. Edit `RootComponents/CMS/cms.js`:

```jsx
import React, {Component, Fragment} from 'react';
import CategoryList from '../../components/CategoryList';
import {Link} from "react-router-dom";

export default class CMS extends Component {
    render() {
        return <Fragment>
            <CategoryList title="Shop by category" id={2} />
            <Link to={'/junk.html'}>Click me</Link>
        </Fragment>;
    }
}
```

2. Click the link in Venia
3. See that the error message reads "That page could not be found. Please try again." instead of "Something went wrong. Please try again."

## Proposed Labels for Change Type/Package

## Checklist:
